recipe/migrations/libprotobuf4251.yaml

```
@@ -0,0 +1,8 @@
+__migrator:
+  build_number: 1
+  commit_message: Rebuild for libprotobuf 4.25.1
+  kind: version
+  migration_number: 1
+libprotobuf:
+- 4.25.1
```

Let's couple this with grpc 1.60
```suggestion
libgrpc:
- "1.60"
libprotobuf:
- 4.25.1
```